numpy/core/src/common/ufunc_override.c

```diff
@@ -97,8 +99,11 @@ PyUFuncOverride_GetOutObjects(PyObject *kwds, PyObject **out_kwd_obj, PyObject *
          * The C-API recommends calling PySequence_Fast before any of the other
          * PySequence_Fast* functions. This is required for PyPy
          */
-        PyObject *seq = PySequence_Fast(*out_kwd_obj, "Could not convert object to sequence");
+        PyObject *seq;
         int ret;
+        Py_INCREF(*out_kwd_obj);
```

Comment:
I don't think this is necessary, and it think it leaks if `seq == NULL`.

I think you should:
* Remove this line
* Add `*out_kwd_obj = NULL` in the error path below
* Replace `Py_SETREF(*out_kwd_obj, seq);` with `*out_kwd_obj = seq`

Reply:
If the calling convention for `PyUFuncOverride_GetOutObjects` is that `*out_kwd_obj` is a borrowed reference, who will decref `seq`?

Reply:
Fixed the missing decref if seq == NULL

Reply:
Ahh, nvrmind. I didn't read your comment correctly. Fixing as per your reccomendation